libs/sdk-backend-tiger/api/sdk-backend-tiger.api.md

```diff
@@ -363,6 +375,7 @@ export type TigerSpecificFunctions = {
     getWorkspaceCustomAppSettings?: (workspaceId: string, applicationName?: string) => Promise<ICustomApplicationSetting[]>;
     createWorkspaceCustomAppSetting?: (workspaceId: string, id: string, applicationName: string, content: object) => Promise<ICustomApplicationSetting>;
     deleteWorkspaceCustomAppSetting?: (workspaceId: string, settingId: string) => Promise<void>;
+    getEntityUser?: (id: string) => Promise<IEntityUser>;
```

Comment:
Can we rather use `IUser` interface from `sdk-model` https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-model/src/user/index.ts as a return type?

Reply:
Thank you @kandl, I have updated the code to use `IUser` as a return type. Thanks!